packages/ui-components/src/ExpandableText/ExpandableText.module.scss

```diff
@@ -0,0 +1,10 @@
+@import "../styles/tokens.scss";
+
+.crl-anchor {
```

Comment:
Since this is a CSS module, you don't need the `crl-` prefix for this classname.